recipes/afwizard/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
```

Comment:
You'll also need to target the channel for your beta build: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Reply:
Specified the `dev` channel target.